typings/index.d.ts

```diff
@@ -191,7 +192,7 @@ export class Giveaway<ExtraData = any> extends EventEmitter {
     public fillInEmbed(embed: unknown): MessageEmbed | null;
     public fillInComponents(components: (MessageActionRow | MessageActionRowOptions)[]): MessageActionRow[];
     public fillInComponents(components: unknown): MessageActionRow[] | null;
-    public exemptMembers(member: GuildMember): Promise<boolean>;
+    public exemptMembers(member: GuildMember, giveaway: Giveaway): Awaitable<boolean>;
```

Comment:
```suggestion
    public exemptMembers(member: GuildMember): Promise<boolean>;
```
this one is the function which executes the exemptMembers function of the user. not the user function itself.

Reply:
coincidently I fixed it 1s ago xD